src/remover.rs

```diff
@@ -17,14 +17,10 @@ pub fn remove_dep(
     let remove_start: usize = search_backwards_non_whitespace(text_start, contents);
     let remove_end: usize = range_to_remove.end().into();
 
-    let new_contents = contents.split_off(remove_start);
-    let end_section = new_contents
-        .chars()
-        .skip(remove_end - remove_start)
-        .collect::<String>();
-    contents.push_str(&end_section);
+    let (before, _) = contents.split_at(remove_start);
+    let after = contents.chars().skip(remove_end).collect::<String>();
```

Comment:
we can avoid an allocation
```suggestion
    let (before, mid) = contents.split_at(remove_start);
    let (_, after) = mid.split_at(remove_end + text_start - remove_start);
```